var/spack/repos/builtin/packages/py-python-rapidjson/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPythonRapidjson(PythonPackage):
+    """Python wrapper around rapidjson."""
+
+    homepage = "https://github.com/python-rapidjson/python-rapidjson"
+    url      = "https://pypi.io/packages/source/p/python-rapidjson/python-rapidjson-0.9.1.tar.gz"
+
+    version('0.9.1', sha256='ad80bd7e4bb15d9705227630037a433e2e2a7982b54b51de2ebabdd1611394a1')
+
+    depends_on('python@3.4:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+
+    depends_on('py-readme-renderer@24.0', type=('build', 'run'))
+    depends_on('py-twine@1.13.0', type=('build', 'run'))
+    depends_on('py-simplejson@3.16.0', type=('build', 'run'))
+    depends_on('py-ujson@1.35', type=('build', 'run'))
+    depends_on('py-yajl@0.3.5', type=('build', 'run'))
```

Comment:
I would remove all of these dependencies aside from python/setuptools. If you look at the [Conda-Forge recipe](https://github.com/conda-forge/python-rapidjson-feedstock/blob/master/recipe/meta.yaml), `python-rapidjson` doesn't need any of these dependencies. They are only used for development/benchmarking.

Reply:
updated. It's strange how the Conda-Forge and github dependencies seem to differ, but it does build OK without these.

Reply:
Don't let `requirements.txt` trick you, those aren't the package dependencies. They are just a list of tools the developers use, including for unit tests (pytest) and uploading new releases (twine). The only trustworthy places to check for dependencies are `setup.py`, `setup.cfg`, and `pyproject.toml`.